cmd/lib/runfile/cmd.go

```diff
@@ -279,6 +285,29 @@ func setUpBQLStmt(tb *bql.TopologyBuilder, bqlFile string) error {
 	return nil
 }
 
+func checkStates(tb *bql.TopologyBuilder, saveUDSList string) error {
```

Comment:
If the only invalid condition is that the topology doesn't have a UDF specified by the argument, `hasStates` may be better?